pages/3_managemenu.php

```diff
@@ -1,17 +1,18 @@
 <?php
-
+//Set the title of the page.
 setTitle("Beheren menu");
 
-//If requested, remove the provided dishes.
-if (isset($_POST['delete']) && isset($_POST['dishesToRemove'])) {
+//If requested, change the status of the dishes.
+if (isset($_POST['switch_status']) && isset($_POST['dishesToRemove'])) {
     foreach($_POST['dishesToRemove'] as $iddish){
+        base_query("UPDATE dish SET ");
         base_query("DELETE FROM dish WHERE Id = :dishid", [':dishid'=>$iddish]);
 
     }
 }
 
 //If requested, remove the provided dishes, only when a category is empty (no dishes or subcategory).
-if(isset($_POST['delete']) && isset($_POST['categoriesToRemove'])){
+if(isset($_POST['switch_status']) && isset($_POST['categoriesToRemove'])){
     foreach($_POST['categoriesToRemove'] as $idcategory){
         base_query("DELETE FROM dishcategory WHERE Id = :categoryid", [':categoryid'=>$idcategory]);
```

Comment:
same als de eerder genoemde delete, word de status nu de-actief? of word die nog steeds verwijderd?